crates/bevy_ui/src/focus.rs

```diff
@@ -166,12 +204,352 @@ pub fn ui_focus_system(
             FocusPolicy::Pass => { /* allow the next node to be hovered/clicked */ }
         }
     }
-    // reset lower nodes to None
-    for (_entity, _focus_policy, interaction, _) in moused_over_z_sorted_nodes {
-        if let Some(mut interaction) = interaction {
-            if *interaction != Interaction::None {
-                *interaction = Interaction::None;
+}
+
+trait CursorResource: Resource {
```

Comment:
I really dislike the use of the generic plus trait here.

Reply:
Ah I see: it's to work around the difficulty mocking `Windows`?

Reply:
See my other comment - how else would I make `Windows` do what I want in a unit test? Everything I tried ended up with a dependency on hardware.